### DISCUSSION TITLE
Intended approach to data within a CRUD application

### DISCUSSION THREAD

**Discussion by then3rdman** [nuxt/nuxt#16724] [General]
Understandably the documentation for Nuxt 3 is a little undercooked. So it is not entirely clear what the intent of the framework is for handling data within a typic CRUD application.

`useFetch` makes a lot of senes for the case if a simple data fetch (`GET` request). However because a little clunky when we consider using it for a `POST` or `PATCH`.  Particularly given the fact there is no way to change the body of the request once the inital request is made.

I don't really want to go too far into the details of my thoughts but rather ask specifically, what is the INTENDED approach? Would love to get some input from the framework team.

**Comment by TheAlexLichter** [comment]
From my point of view, `useFetch` is more for data fetching. Sending data to the backend itself is better via [`$fetch`](https://v3.nuxtjs.org/api/utils/$fetch#fetch).

**Comment by then3rdman** [comment]
Yeh I have come to the same conclusion.

I still would love a team member to chime in with their overall view of data handling within the framework
